specification/compute/resource-manager/Microsoft.Compute/stable/2020-09-30/disk.json

```diff
@@ -0,0 +1,3205 @@
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/DiskAccessNameParameter"
+          },
+          {
+            "$ref": "#/parameters/PrivateEndpointConnectionNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/PrivateEndpointConnection"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get information about a private endpoint connection under a disk access resource.": {
+            "$ref": "./examples/GetInformationAboutAPrivateEndpointConnection.json"
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "DiskAccesses"
+        ],
+        "operationId": "DiskAccesses_DeleteAPrivateEndpointConnection",
+        "description": "Deletes a private endpoint connection under a disk access resource.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/DiskAccessNameParameter"
+          },
+          {
+            "$ref": "#/parameters/PrivateEndpointConnectionNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK"
+          },
+          "202": {
+            "description": "Accepted"
+          },
+          "204": {
+            "description": "If the private endpoint connection is already deleted, this is an expected error code."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Delete a private endpoint connection under a disk access resource.": {
+            "$ref": "./examples/DeleteAPrivateEndpointConnection.json"
+          }
+        },
+        "x-ms-long-running-operation": true
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/restorePointCollections/{restorePointCollectionName}/restorePoints/{vmRestorePointName}/diskRestorePoints/{diskRestorePointName}": {
```

Comment:
Where are all the APIs related to restorePointCollections and restorePoints?  How does a user discover these deeply nested resources with LISTs and GETs at each level?

Reply:
restorePointCollections and restorePoints are created by CRP via privilege calls. The API for createing restorePoints(AllocateRestorePoints) will return the diskRestorePoint armId in response body to client.

Reply:
What about after creation time? Are there collection gets for restorePointCollections and  restorePointCollections/restorePoints? I didn't find any in other swaggers or this PR. Without those, it seems pretty hard for clients to discover that restorePointCollections contain restorePointCollections/restorePoints (maybe that's somewhat intuitive), and that restorePointCollections/restorePoints contain restorePointCollections/restorePoints/diskRestorePoints.

Reply:
What about after creation time? Are there collection gets for restorePointCollections and restorePointCollections/restorePoints? I didn't find any in other swaggers or this PR. Without those, it seems pretty hard for clients to discover that restorePointCollections contain restorePointCollections/restorePoints (maybe that's somewhat intuitive), and that restorePointCollections/restorePoints contain restorePointCollections/restorePoints/diskRestorePoints.

Reply:
What about after creation time? Are there collection gets for restorePointCollections and restorePointCollections/restorePoints? I didn't find any in other swaggers or this PR. Without those, it seems pretty hard for clients to discover that restorePointCollections contain restorePointCollections/restorePoints (maybe that's somewhat intuitive), and that restorePointCollections/restorePoints contain restorePointCollections/restorePoints/diskRestorePoints.

Reply:
After creation time, CRP has APIs that return restorePointCollections/restorePoints here link below. 

https://msazure.visualstudio.com/One/_git/Compute-CPlat-Core?path=%2Fsrc%2FCRP%2FDev%2FComputeResourceProvider%2FService%2FAPI%2FControllers%2FRestorePointCollectionsControllerBase.cs&version=GBmaster&line=34&lineEnd=39&lineStartColumn=9&lineEndColumn=79&lineStyle=plain&_a=contents